Sources/Stencil/Variable.swift

```diff
@@ -110,16 +110,16 @@ public struct Variable: Equatable, Resolvable {
       return resolve(bit: bit, collection: array)
     } else if let string = context as? String {
       return resolve(bit: bit, collection: string)
+    } else if let value = context as? DynamicMemberLookup {
+      return value[dynamicMember: bit]
     } else if let object = context as? NSObject {  // NSKeyValueCoding
-      #if os(Linux)
+      #if !canImport(ObjectiveC)
```

Comment:
Tiny nitpick: I'd switch these around, so have the "positive" case first, as we do something there. I know that you didn't change the order, it's just something we can correct now.

Reply:
Extra note: @compnerd mentioned that `canImport` isn't a sufficient check, so we'll probably need that `_runtime` check, or hopefully something more "official" than that.

Reply:
Hello @djbe ,

thank you for your comment.

I would like to address your points:

> I'd switch these around, so have the "positive" case first

Totally making sense, I missed this, and will apply this change right afterwards.

> Extra note: @compnerd mentioned that canImport isn't a sufficient check, so we'll probably need that _runtime check, or hopefully something more "official" than that.

As far as I have tried, using the (deprecated) `swift test --generate-linuxmain` generates `LinuxMain.swift` and a list of `XCTestManifests.swift` files, where this check is included:

```swift
#if !canImport(ObjectiveC)
import XCTest
```

So I would rather stick with something SPM engineering decided to go with, rather than being extra safe. Not only that, it is undefined if `_runtime` will still work in Swift 6.0 or future versions since it is internal.

Reply:
I know, not a fan of using internal Swift functionality. We discussed it a bit further in there, and the current conclusion is that `canImport` is sufficient.